lib/jsmidgen.js

```diff
@@ -368,7 +368,13 @@ var Midi = {};
 
 	var File = function(config){
 		if (!this) return new File(config);
+
 		var c = config || {};
+		if (c.ticks && typeof c.ticks !== 'number' || c.ticks <= 0 || c.ticks >= (1 << 15)) {
```

Comment:
I'd probably do:

```
if (c.ticks) {
  if (typeof c.ticks !== 'number') {
    throw new Error('Ticks per beat must be a number');
  }
  if (c.ticks <=0 || c.ticks >= (1 << 15)) {
    throw new Error('Ticks per beat must be between 1 and 32767');
  }
  this.ticks = Math.floor(c.ticks);
} else {
  this.ticks = 128;
}
```

in order to have clear error messages for different types of errors... and also to ensure that the number of ticks is an integer.